content/blog/configure-isut-and-ams-for-vlcm-based-firmware-updates-on-gen-12-servers.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to Configure iSUT and AMS for vLCM-Based Firmware Updates on HPE
+  Gen12 Servers (High Security Mode)
+date: 2025-10-01T02:30:17.312Z
+author: Vijayakannan M
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - OV4VC
+  - VLCM
+  - COM4VC
+  - HPE OneView
```

Comment:
If you want to tie your blog post to the [HPE Oneview landing page](https://developer.hpe.com/platform/hpe-oneview/home/) on the HPE Dev community portal, please add the **individual tag** below:

hpe-oneview